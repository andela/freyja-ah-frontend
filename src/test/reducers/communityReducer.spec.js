import reducer, { initialState } from '../../store/reducers/communityReducer';

describe('reset password reducer', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });

  it('should handle LOADING', () => {
    expect(reducer(
      initialState,
      {
        type: 'LOADING',
      },
    )).toMatchSnapshot();
  });
  it('should handleGET_COMMUNITY_MESSAGES', () => {
    expect(reducer(
      initialState,
      {
        type: 'GET_COMMUNITY_MESSAGES',
      },
    )).toMatchSnapshot();
  });
  it('should handle GET_COMMUNITY_MESSAGES_ERROR', () => {
    expect(reducer(
      initialState,
      {
        type: 'GET_COMMUNITY_MESSAGES_ERROR',
      },
    )).toMatchSnapshot();
  });
});
