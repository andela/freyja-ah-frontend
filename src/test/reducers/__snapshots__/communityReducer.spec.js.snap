// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reset password reducer should handle DELETE_ERROR 1`] = `
Object {
  "allMessages": Array [],
  "deleteError": undefined,
  "deleteSuccess": "",
  "error": null,
  "isLoading": false,
}
`;

exports[`reset password reducer should handle DELETE_SUCCESS 1`] = `
Object {
  "allMessages": Array [],
  "deleteError": "",
  "deleteSuccess": undefined,
  "error": null,
  "isLoading": false,
}
`;

exports[`reset password reducer should handle GET_COMMUNITY_MESSAGES_ERROR 1`] = `
Object {
  "allMessages": Array [],
  "deleteError": "",
  "deleteSuccess": "",
  "error": undefined,
  "isLoading": false,
}
`;

exports[`reset password reducer should handle LOADING 1`] = `
Object {
  "allMessages": Array [],
  "deleteError": "",
  "deleteSuccess": "",
  "error": null,
  "isLoading": true,
}
`;

exports[`reset password reducer should handleGET_COMMUNITY_MESSAGES 1`] = `
Object {
  "allMessages": undefined,
  "deleteError": "",
  "deleteSuccess": "",
  "error": null,
  "isLoading": false,
}
`;

exports[`reset password reducer should return the initial state 1`] = `
Object {
  "allMessages": Array [],
  "deleteError": "",
  "deleteSuccess": "",
  "error": null,
  "isLoading": false,
}
`;
